cases/toga/TogaCase.py

```diff
@@ -0,0 +1,19 @@
+# nuitka-project: --standalone
+import toga
+
+def button_handler(widget):
+    print("hello")
+
+def build(app):
+    box = toga.Box()
+    button = toga.Button("Hello world", on_press=button_handler)
+    button.style.padding = 50
+    button.style.flex = 1
+    box.add(button)
+    return box
+
+def main():
+    return toga.App("First App", "org.beeware.helloworld", startup=build)
+
+if __name__ == "__main__":
```

Comment:
What about the NUITKA_LAUNCH_TOKEN thing, to display it or not, that seems like  good idea for coverage of GUI when interactively ran or not.

Reply:
sure, i think it's good enough as it is though, maybe once we have the watch running well enough we can extend the GUI apps in that way